components/ConsumptionGauge.vue

```diff
@@ -59,6 +59,13 @@ export default {
       }
 
       return (this.used * 100) / this.capacity;
+    },
+    amountTemplateValues() {
+      return {
+        used:  this.numberFormatter(this.used || 0),
+        total: this.numberFormatter(this.capacity || 0),
+        unit:  this.displayUnits
+      };
```

Comment:
FYI it's not necessarily applicable here since you have the component provide its own format, but keep in mind the i18n messageformat can do a lot stuff like this (`{used, number, ::.00}`).

https://formatjs.io/docs/icu-syntax/

https://github.com/unicode-org/icu/blob/master/docs/userguide/format_parse/numbers/skeletons.md